shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -57,3 +59,65 @@ void PlatformChannel::HandleMethodCall(
     result->NotImplemented();
   }
 }
+
+// Clipboard constants and variables
+namespace Clipboard {
+
+// naive implementation using std::string as a container of internal clipboard
+// data
+std::mutex is_processing_mutex;
+std::string string_clipboard = "";
+
+static constexpr char kTextKey[] = "text";
+static constexpr char kTextPlainFormat[] = "text/plain";
+static constexpr char kUnknownClipboardFormatError[] =
+    "Unknown clipboard format error";
+static constexpr char kUnknownClipboardError[] =
+    "Unknown error during clipboard data retrieval";
+
+void GetData(
+    const flutter::MethodCall<rapidjson::Document>& call,
+    std::unique_ptr<flutter::MethodResult<rapidjson::Document>> result) {
+  const rapidjson::Value& format = call.arguments()[0];
+
+  // https://api.flutter.dev/flutter/services/Clipboard/kTextPlain-constant.html
+  // API supports only kTextPlain format
+  if (strcmp(format.GetString(), kTextPlainFormat) != 0) {
+    result->Error(kUnknownClipboardFormatError,
+                  "Clipboard API only supports text.");
+    return;
+  }
+
+  {
+    std::unique_lock<std::mutex> lock(is_processing_mutex);
```

Comment:
You are right, I left them because of reworking my previous version of this feature https://github.com/flutter-tizen/engine/pull/65, which includes using asynchronous native API in one of previous versions. After your comment, I removed mutexes.